docs/generate_user_docs.sh

```diff
@@ -30,6 +30,7 @@
 mrtrix_root=$( cd "$(dirname "${BASH_SOURCE}")"/../ ; pwd -P )
 export PATH=$mrtrix_root/bin:"$PATH"
 dirpath=${mrtrix_root}'/docs/reference/commands'
+export LC_ALL=C
```

Comment:
Is this to stop `tcksift` and `tcksift2` being reordered in the docs command list? :crossed_fingers:

Reply:
Yep. I expect you've come across the same issue?